### DISCUSSION TITLE
EasyPanel

### DISCUSSION THREAD

**Discussion by reziamini** [HassanZahirnia/laravel-package-ocean#76] [Package Suggestions]
EasyPanel is a beautiful, customizable and flexible admin panel based on Livewire for Laravel.

https://github.com/rezaamini-ir/laravel-easypanel

Category: CMS & Admin Panels

**Comment by HassanZahirnia** [comment]
Hey! Thanks for your suggestion.

I see your package has not been updated for a long time, and doesn't seem be compatible with Laravel 10. I would only add packages that are actively maintained and up to date with latest Laravel versions.

If you attend to updating it more often I'll be happy to add it to the database!
